### PR TITLE
Added question page content and exit page content (for all no answers)

### PR DESCRIPTION
**Pull request by bnaolu**
Added content for issues: "Question page..." and "Exit page content (for all no answers)".

### CODE REVIEW

**Comment by bnaolu** [comment]
- Added content for question pages
- Added content for exit pages
   - back buttons do not link back to previous pages, yet (still working on this)

Let me know your thoughts on how this is looking.

**Comment by pglevy** [comment]
Don't worry about the links. That will be handled with the JS logic.

File: _data/questions-yn.yml [hunk 1]
```diff
@@ -1,36 +1,36 @@
 - number: 1
-  text: Who did this?
+  text: Is your complaint about a covered entity?
```
Comment (pglevy) [inline _data/questions-yn.yml]: It might get wordier, but how can we not say "covered entity?"

File: _includes/radio.html [hunk 2]
```diff
@@ -11,7 +11,7 @@ <h1>
     {% for answer in item.answers %}
     <div class="usa-radio">
         <input class="usa-radio__input usa-radio__input--tile" id="{{ item.slug }}{{ forloop.index }}" type="radio" name="{{ item.slug }}" value="{{ answer.path }}">
-        <label class="usa-radio__label" for="{{ item.slug }}{{ forloop.index }}">{{ answer.text }}</label>
+        <label class="usa-radio__label margin-bottom-6" for="{{ item.slug }}{{ forloop.index }}">{{ answer.text }}</label>
```
Comment (pglevy) [inline _includes/radio.html]: This utility class doesn't seem to have any effect. What are you trying to accomplish?

File: _layouts/exit-when.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% include header.html %}
```
Comment (pglevy) [inline _layouts/exit-when.html:1]: Do we need a separate layout for this?

File: _layouts/exit-who.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    {% include step-indicator.html %}
+    {{ content }}
+    <div class="usa-accordion">
+  
+      <!-- Use the accurate heading level to maintain the document outline -->
+      <h4 class="usa-accordion__heading">
+        <button class="usa-accordion__button"
+          aria-expanded="true"
+          aria-controls="a1">
+          Law Enforcement Agency
```
Comment (pglevy) [inline _layouts/exit-who.html]: We should separate the content out from this template. I can help with this if needed. Or you can look at the FAQ page template. I did something similar there.

File: _layouts/question-include.html [hunk 5]
```diff
@@ -8,7 +8,7 @@
     {% else %}
         This is not a cake nor a cookie
     {% endcase %}
-    <a href="#back" id="backButton" class="usa-button usa-button--outline">Back</a> <a href="#next" id="nextButton" class="usa-button">Next</a>
+    <a href="#back" id="backButton" class="usa-button usa-button--outline margin-top-3">Back</a> <a href="#next" id="nextButton" class="usa-button">Next</a>
```
Comment (pglevy) [inline _layouts/question-include.html]: I did some other stuff with placement of buttons, so we probably don't need this.

File: exit-what-1.md [hunk 6]
```diff
@@ -3,6 +3,7 @@ layout: default
 title: Exit What 1
 ---
 
-# Exit what 1
+# Things to know:
 
-Explanation of why not to proceed after the first what question.
+- If you are requesting access to a copy of your health insurance information, your health plan or provider has up to 30 days (sometimes more) to provide the information. Your health insurance plan or provider may require that you make your request in writing.
+- Learn more about your right to access your medical record [here](https://www.hhs.gov/hipaa/for-individuals/medical-records/index.html).
```
Comment (pglevy) [inline exit-what-1.md]: Let's make this whole sentence a link (without the period), instead of just "here." In general, avoid linking just the word "here."

File: exit-what-1.md [hunk 7]
```diff
@@ -3,6 +3,7 @@ layout: default
 title: Exit What 1
 ---
 
-# Exit what 1
+# Things to know:
```
Comment (pglevy) [inline exit-what-1.md]: Can we make this heading more descriptive? Maybe something like "Timeline for requesting your health information"?

File: exit-what-2.md [hunk 8]
```diff
@@ -3,6 +3,6 @@ layout: default
 title: Exit What 2
 ---
 
-# Exit what 2
+# Unfortunately, we can’t complete your request.
 
-Explanation of why not to proceed after the second what question.
+The Department of Health and Human Service’s works only with health information-related violations pertaining to HIPAA.
```
Comment (pglevy) [inline exit-what-2.md]: This is a dead end. Is there anything else we can provide them here in terms of next steps? Or at least clarifying (or restating) why we don't think it applies. Let's try to avoid just referring to "HIPPA" by itself and assume that people know that that means.

File: exit-when.md [hunk 9]
```diff
@@ -1,8 +1,10 @@
...
-layout: default
+layout: exit-when
 title: Exit When
 ---
 
-# Exit when
+# You may still be able to file a complaint.
 
-Explanation of why not to proceed after the when question.
+Please note that under our regulations, we normally only accept complaints that are filed within 180 days of when a complainant knew or should have known that the alleged violation occurred.
```
Comment (pglevy) [inline exit-when.md]: "complaintant"

File: question-1.md [hunk 10]
```diff
@@ -5,7 +5,11 @@ number: 1
...
-- example 2
\ No newline at end of file
+- Health care providers
<br>
+(Doctors, clinics, psychologists, dentists, chiropractors, nursing homes, pharmacies)
+-  Health plans
<br>
+(Health insurance companies, HMOs, company health plans, government health programs such as Medicare, Medicaid, and the military and veterans health care programs)
+- Health care clearinghouse
<br>
```
Comment (pglevy) [inline question-1.md:14]: I'm still not sure what this is. We may need to provide more information or completely reword this to avoid the word "clearinghouse." Probably makes people think of "sweepstakes," which is probably incorrect and not helpful. :)

File: question-3.md [hunk 11]
```diff
@@ -3,4 +3,15 @@ layout: question-include
 title: Question 3
 number: 3
 type: radio
----
\ No newline at end of file
+---
+
+**Examples of health information rights::**
```
Comment (pglevy) [inline question-3.md]: Extra colon

File: question-3.md [hunk 12]
```diff
@@ -3,4 +3,15 @@ layout: question-include
...
+
+- [HIPAA Privacy Rule
](https://www.hhs.gov/hipaa/for-individuals/guidance-materials-for-consumers/index.html?language=es)<br>
```
Comment (pglevy) [inline question-3.md]: This copy works pretty well, but in general, I think it would be more helpful to lead with the explanation. Most people aren't to click or mentally process these official names. But they will understand "Federal law recognizes your rights over your health information and set rules and limits on who can see and receive it." So maybe reversing them and following with the link in parentheses?

**Review by pglevy — CHANGES_REQUESTED** [review]
Lots of great work here. Nice job! I left a potpourri of comments in individual files. We can sync up if you have questions or need to go into more detail.

File: _includes/radio.html (hunk 2 above)
Comment (bnaolu) [reply _includes/radio.html]: LOL absolutely nothing! I was getting lost in all the files! Thank you for catching this, I'll have to be more careful next time.

File: _layouts/exit-when.html (hunk 3 above)
Comment (bnaolu) [reply _layouts/exit-when.html:1]: Hmm not sure; what's your opinion on that?

**Comment by bnaolu** [comment]
Changes made:
- restructured exit-who.html content to better accommodate for accordion feature
   - added content for exit-who.html resources 
- fixed code syntax
   - removed unnecessary button margin class
- fixed sentence wordings throughout

File: _layouts/exit-when.html (hunk 3 above)
Comment (pglevy) [reply _layouts/exit-when.html:1]: I'll look again once it's merged.

**Review by pglevy — APPROVED** [review]
